doc/technotes/localeModels.rst

```diff
@@ -21,13 +21,14 @@ multiple instances of memories and/or processors with differing
 characteristics.  They may also be hierarchical, with parent locales
 containing one or more child sublocales within them.
 
-There are currently two locale models available, flat and NUMA.  The
+There are currently three locale models available, flat, NUMA, and KNL.  The
 flat model is the default and maps closely to the view of locales
 implemented in the 1.7 release.  The NUMA locale model maps sublocales
 to NUMA domains.  The NUMA model is currently implemented at a prototype
 level. Performance has not yet been a focus in the NUMA locale model and
 will require additional effort in future releases.  We expect to add
-more locale models in future releases.
+more locale models in future releases.  The KNL locale model provides
+access to on-chip high-bandwidth DRAM.
```

Comment:
I would swap these last two sentences, so that we end with the "and more are coming in the future" one.  Also, the KNL sentence should say that it's for compute nodes based on self-hosted Intel Xeon Phi (formerly Knights Landing) processors and includes provision for access to the tightly-coupled on-board high bandwidth DRAM memory.  (Strictly speaking the HBM isn't on-chip, it's on-package, but I think they're just referring to it using "on-board" and other general phrases like that so they have implementation flexibility.